src/module/actor/monster-sheet.js

```diff
@@ -224,7 +225,7 @@ export class OseActorSheetMonster extends OseActorSheet {
         .index.filter((el) => el._id === data.id);
       link = `@Compendium[${data.pack}.${data.id}]{${tableData[0].name}}`;
     } else {
-      link = `@RollTable[${data.id}]`;
+      link = `@UUID[${data.uuid}]`;
```

Comment:
Can anyone tell me if this works in v9?

Reply:
this is a V10 breaking change

https://foundryvtt.com/article/v10-text-editor/#uuidLinks
<img width="1164" alt="image" src="https://user-images.githubusercontent.com/1410433/194965234-7391c3e0-c44e-4914-a024-b7499574a21d.png">

Reply:
thanks for noting it specifically. This will go on the pile of things earmarked "would need to be changed if V9 backwards compatibility once again becomes a priority"

Reply:
Sure thing. I can change this to use the line I'm removing if the user's on v9, and the line I'm adding if the user's on v10, if we like.

Reply:
While plenty of the codebase attempts V9-10 cross-compatibility, we are still overall pretty far away from achieving it. I'm planning to accept this PR as-is :)